fe/fe-core/src/test/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MvRewriteOptimizationTest.java

```diff
@@ -1499,7 +1507,7 @@ public void testUnionRewrite() throws Exception {
 
         String query5 = "select * from multi_mv_1";
         String plan5 = getFragmentPlan(query5);
-        PlanTestBase.assertContains(plan5, "multi_mv_1", "multi_mv_2", "multi_mv_3", "UNION");
+        PlanTestBase.assertContains(plan5, "multi_mv_1", "multi_mv_2", "UNION");
```

Comment:
why the final plan does not contain "multi_mv_3" ?

Reply:
`multi_mv_3 ` can be covered by `multi_mv_2`. this is done by creating mv scan operator with partition filters.